.config-schema.json

```diff
@@ -56,6 +56,16 @@
             "default": 1000,
             "x-env-variable": "OPENFGA_LIST_OBJECTS_MAX_RESULTS"
         },
+        "requestDurationDatastoreQueryCountBuckets": {
+            "description": "Datastore query count buckets used in labelling request duration by request histogram statistics",
+            "type": "array",
+            "items": {
+                "minimum": 0,
```

Comment:
Isn't the minimum 1 if you provide the config?

If you omit the config (e.g. if the config is nil) then the defaults are used, but otherwise you have to provide at least 1 bucket.

Reply:
I do not understand what you mean.  Minimum 0 means you cannot put a negative number for the bucket (i.e., there will never be datastore query count of -1).  It does not imply the size of the array.

I think what you want is "minItems": 1,

Reply:
Oh, yes. `minItems: 1` but each item's minimum can be no smaller than 0. Yep.